docs/features/chord_keyboard.md

```diff
@@ -21,56 +19,93 @@ To access the Chord Keyboard Layout:
+ 
+Each row starts from the next scale degree up or down, and then moving across the pads, the intervals are the following in scale intervals (if a 3rd up from a scale degree is a major 3rd, it will be a major 3rd in the row, if it is a minor 3rd, it will be a minor 3rd in the row).
+  - Root, 5th, 3rd + Octave (Compound 3rd), 7th + Octave (Compound 7th), 5th + Octave (Compound 5th), 3rd + 2 * Octave (Double Compound 3rd), 2nd + 2 * Octave (Double Compound 2nd), 6th + Octave (Compound 6th), Octave, 5th, 7th, 3rd - Octave, 2nd
+  - The final column is a triad based on the Root of the row with 5th, Compound 3rd above it.
+  
+**As the Chord Keyboard is still in development, the intervals chosen for each row are not final and may change in the future. We welcome feedback on the intervals chosen for each row.**
+    
 
-- **Modes:**
-  - **Row Mode:** Inspired by the Launchpad Pro, this mode lays out chords horizontally across the pads, with generally simpler intervals on the left and more complex ones on the right. 
-  - **Column Mode:** This mode arranges harmonically similar chords vertically. Chords in the same column can often be substituted for each other to explore different harmonic flavors. As an example, if your in a major scale, the first column will contain various option for the I chord, the second column will contain various options for the ii chord, and so on. 
+### Color Coding
 
-- **Color Coding:**
-  - Chords are colored based on their quality:
-    - Major: Blue
-    - Minor: Purple
-    - Dominant: Cyan
-    - Diminished: Green
-    - Augmented: Greyish Blue
-    - Other: Yellow
+In Column mode, Chords are colored based on their quality, and in Row mode, the Rows are based on the quality of the chord from that scale degree:
 
-- **Controls:**
-  - **Embedded Controls:**
-    - Because the Chord Keyboard has more features than other keyboard layouts, embedded into the right couple columns of the main grid is are controls. Currently it includes controls for switching between Row Mode and Column Mode, with more features planned for the future.
-    - At the top of the right most main grid column, you can select the current mode (`ROW` and `COLUMN`) by pressing the corresponding pad. It defaults to Row Mode.
-      - In addition, you can use `SHIFT` + `◀︎▶︎` to also cycle through the `ROW` and `COLUMN` mode.
-  - turning either `◀︎▶︎` or  `▼︎▲︎` will move up or down by scale degrees.
+  - Major: Blue
+  - Minor: Purple
+  - Dominant: Cyan
+  - Diminished: Green
+  - Augmented: Greyish Blue
+  - Other: Yellow
 
-##### Scale Considerations
+### Controls
+
+turning either `◀︎▶︎` or  `▼︎▲︎` will move up or down by scale degrees.
+
+#### Embedded Controls:
+
+Because the Chord Keyboard has more features than other keyboard layouts, embedded into the right couple columns of the main grid is are controls. Currently it includes controls for switching between Row Mode and Column Mode, with more features planned for the future.
```

Comment:
```suggestion
Because the Chord Keyboard has more features than other keyboard layouts, embedded into the right couple columns of the main grid are controls. Currently it includes controls for switching between Row Mode and Column Mode, with more features planned for the future.
```